bundle-workflow/src/test_workflow/put_file_into_S3.py

```diff
@@ -0,0 +1,361 @@
+        parser.add_argument("-a", "--assume_role", nargs="?", default="os.getenv('OPENSEARCH_TESTING_ROLE')", help="Assume role")
+        parser.add_argument("-s", "--session", nargs="?", default="get-files-session", help="Session name")
+        parser.add_argument("-b", "--bucket_name", help="Bucket name", required=True)
+        parser.add_argument("-o", "--object_path", help="ObjectPath in Bucket", required=True)
+        parser.add_argument("-f", "--file_path", help="Filepath of the result file", required=True)
+
+        args = parser.parse_args()
+
+        return args
+
+    def conversion_to_html(self, args):
+        with open(args.file_path + args.result_file) as json_data:
+            try:
+                data = json.load(json_data)
+                testExecutionId = data['testExecutionId']
+                filename = "%s.html" % testExecutionId
+                values = {1: 'opCount', 2: 'opErrorCount', 3: 'opErrorRate', 4: 'p0', 5: 'p50', 6: 'p100', 7: 'p50',
+                          8: 'p90',
+                          9: 'p99', 10: 'p100'}
+
+                stored = {1: '1', 2: '2', 3: '3', 4: '4', 5: '5', 6: '6', 7: '7', 8: '8', 9: '9', 10: '10'}
+
+                testParameters = {1: 'testExecutionId', 2: 'testOwner', 3: 'testStartTime', 4: 'testEndTime',
+                                  5: 'testDuration'}
+
+                sut_Parameters = {1: 'dataNodeCount', 2: 'masterNodeCount', 3: 'dataNodeInstanceType',
+                                  4: 'masterNodeInstanceType', 5: 'elasticsearchDistributionType', 6: 'aes',
+                                  7: 'clusterName', 8: 'awsAccount', 9: 'endpoint',
+                                  10: 'region'}
+                work_config = {1: 'dataset', 2: 'warmupIterations', 3: 'testIterations'}
+
+                def testResults_OS(arg1):
+                    if arg1 == 'index':
+                        arg1 = 'index'
+                    else:
+                        arg1 = 'query'
+
+                    for i in range(1, 11):
+                        if i == 4 or i == 5 or i == 6:
+                            val = data['testResults']['operationsSummary'][arg1]['requestsPerSecond'][values[i]]
+                            stored[i] = val
+                            print(stored[i])
+                        elif i == 7 or i == 8 or i == 9 or i == 10:
+                            val = data['testResults']['operationsSummary'][arg1]['latencyMillis'][values[i]]
+                            stored[i] = val
+                            print(stored[i])
+                        else:
+                            val = data['testResults']['operationsSummary'][arg1][values[i]]
+                            stored[i] = val
+                            print(stored[i])
+                    return stored
+
+                def testResults_Operations(arg1):
+                    if arg1 == 'auto-date-histogram':
```

Comment:
`Code Smell: Many If/Else statements that repeat a the value matches allow assignment`
Valid values should be in a enum or array that is easily modified as this list will likely expand over time.  That said why does `index-append` need to be treated differently?

Even better rather than depending on detection of a value to change behavior can you check for a type of data and then shape the behavior according to the data?